db/seeds.rb

```diff
@@ -75,6 +76,18 @@
     # Create animals and a cohort per facility
     cohort = Cohort.find_or_create_by(name: "#{enclosure.name} cohort", enclosure: enclosure, organization: org)
 
+    dead_animal = Animal.find_or_create_by(
```

Comment:
Note to self: we may want to extract this as an option to the animal factory later.